src/R/Components/Impl/Application/Configuration/IO/ConfigurationSettingsWriter.cs

```diff
@@ -29,16 +30,18 @@ public sealed class ConfigurationSettingsWriter : IConfigurationSettingsWriter {
 
         /// <summary>
         /// Persists settings into R file. Settings are written
-        /// into R file as assignment statements such as 'name &lt;- value'.
+        /// into R file as assignment statements such as 'settings$name &lt;- value'.
         /// Value
         /// </summary>
         public void SaveSettings(IEnumerable<IConfigurationSetting> settings) {
             WriteHeader();
+            _writer.WriteLine("settings <- new.env()");
```

Comment:
Good point. How about just using `as.environment(list())` to get a blank new environment that's not connected to anything else, and then using the existing approach to fill it?